index.js

```diff
@@ -37,7 +42,24 @@ function getCartTotal(cart) {
 */
 function filterProductsByPriceRange(products, min, max) {
   const result = [];
+
+  if (!products.length) {
+    throw "No products scanned";
+  }
+  if (typeof min !== "number" || typeof max !== "number") {
+    throw "Price invalid";
+  }
+  if (min > max) {
+    throw "Price invalid";
+  }
+  if (min <= 0 || max <= 0) {
```

Comment:
```suggestion
  if (min < 0 || max <= 0) {
```
It's true that you wouldn't normally find something that's FREE (aka $0) while shopping, but you should allow for 0 as a minimum value.